docs/source/models.rst

```diff
@@ -584,6 +584,42 @@ produce a dataframe, a vector or a list with the predictions as output.
 
 This flavor requires R to be installed in order to be used.
 
+Crate usage
+~~~~~~~~~~~~~~~~~~~~
+
+For a minimal crate model, an example configuration for the predict function is:
+
+.. code-block:: r
+
+    library(mlflow)
+    library(carrier)
+    # Load iris dataset
+    data("iris")
+
+    # Learn simple linear regression model
+    model <- lm(Sepal.Width~Sepal.Length, data = iris)
```

Comment:
To run this example, do we need to define `lm`?

Reply:
@sn8k2s Let's say we have code like this:

```r
print(a)
```


To run this code, we need to define `a` like:

```
a <- "a"
print(a)
```

That's what I meant by `define`.

Reply:
I'm not familiar with R and wondering where `lm` comes from.

Reply:
` lm` come from the package `stats `which is the R stats package.